src/dndbeyond/content-scripts/encounter.js

```diff
@@ -22,6 +25,23 @@ function documentModified(mutations, observer) {
     character.parseStatBlock(monster);
 }
 
+function updateCombatTracker() {
+    if ($(".turn-controls__next-turn-button").length) {
+        $(".ui-dialog-buttonpane").hide(); // get rid of turn controls, to make it more obvious to do things from DDB.
+        const combat = Array.from($(".combatant-card.in-combat")).map(x => [
+            $(".combatant-summary__name", x).text(),
+            $(".combatant-card__initiative-value", x).text(),
+            $(x).hasClass("is-turn"),
+        ]);
```

Comment:
I see what you do, but it's a little difficult to read. I suggest naming it `combatant` instead of `x` and using `$(x).find(selector)` instead of `$(selector, x)`
I also know that you probably used an array like this because I also did that in other parts of the code (when I was still using python's tuple mentality), but I think it would be much better and easier to maintain now if we use an object.

How about : 
```js
const combatants = $(".combatant-card.in-combat")
const combat = combatants.toArray().map(combatant => {
  combatant = $(combatant)
  return {
    name: combatant.find(".combatant-summary__name").text(),
    initiative: combatant.find(".combatant-summary__name").text(),
    turn: combatant.hasClass("is-turn")
  };
});
```